packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,531 @@
+  </CalloutText>
+</Callout>
+
+### Paste components
+
+<Box
+  as="span"
+  display="inline-block"
+  width="auto"
+  marginBottom="space50"
+  paddingTop="space20"
+  paddingBottom="space20"
+  paddingLeft="space30"
+  paddingRight="space30"
+  borderStyle="solid"
+  borderWidth="borderWidth20"
+  borderColor="colorBorderLighter"
+  borderRadius="borderRadius20"
+>
+<Stack as="span" orientation="horizontal" spacing="space20">
+  <Tooltip text="There is ongoing maintenance and net new work in this project. Libraries in this project are at parity with code.">
+    <Anchor><SuccessIcon decorative={false} title="Success icon" color="colorTextSuccess"/></Anchor>
+  </Tooltip>
+  <Text as="p">
+    Libraries in this project are actively maintained
+  </Text>
+</Stack>
+</Box>
+
+The `Paste components` project contains libraries that represent components that are available in Paste. For example,
+within the `Paste components` project, we have a Sketch library called `button`, which has all variants and states of
+our button component in both the Console and Default themes.
+
+#### Linking the libraries
+
+To add Paste components to your designs, you will need to link the relevant component libraries from the `Paste components`
+project in Abstract. There are many libraries available in this project: one library per component available in Paste
+(except for a few [special cases](#special-cases)).
+
+You can link all libraries within `Paste components` if you’d like, or you can link them as you need them. For example, if you are
+working on a flow that uses anchor, button, and card from Paste, at a minimum, you should link those libraries to your project.
+
+![Modal in Abstract where you can select libraries from the Paste compenents project to link to your project](../../../assets/images/design-guidelines/link-component-library-modal.png)
+
+#### Using the libraries
+
+Once you have them linked, you will be able to insert symbols from those libraries into your Sketch files in that Abstract project.
+Use the symbols that match the theme you are using in your project.
+
+![Menu in Sketch where you can navigate through symbols from the Anchor Sketch library](../../../assets/images/design-guidelines/paste-components-symbol.png)
+
+<Callout variant="warning">
+  <CalloutTitle>
+    A note on SendGrid components
+  </CalloutTitle>
+  <CalloutText>
+    You will see that some symbols are provided in the SendGrid theme. If you plan to use them, please reach out to us in #help-design-system.
+  </CalloutText>
+</Callout>
+
+#### Special cases
+
+Most library names in the `Paste components` project mirror the list of components you will find on the docs site, with a few notable
+exceptions: `typography`, `forms`, and `grid and spacing`.
+
+<Box marginBottom="space70">
+<Stack orientation="vertical" spacing="space60">
+<Disclosure>
```

Comment:
Did you try contained disclosures for these, and the ones under "Other libraries"? The lack of a left margin on the disclosure content makes it a little hard to tell where the boundaries between disclosure sections are, but I'm not sure if the contained disclosure would add too much noise.

Reply:
Leaving as-is based on our Slack convo